src/commands/app/use.js

```diff
@@ -76,44 +330,92 @@ class Use extends BaseCommand {
       interactive = false
     }
 
-    const { values: config } = loadAndValidateConfigFile(filePath)
+    // before importing the config, first extract the service api key id
+    const { values: config } = loadAndValidateConfigFile(consoleConfigFileOrBuffer)
     const project = config.project
     const jwtConfig = project.workspace.details.credentials && project.workspace.details.credentials.find(c => c.jwt)
     const serviceClientId = (jwtConfig && jwtConfig.jwt.client_id) || ''
     const extraEnvVars = { [SERVICE_API_KEY_ENV]: serviceClientId }
 
-    return importConfigJson(filePath, process.cwd(), { interactive, overwrite, merge }, extraEnvVars)
+    await importConfigJson(consoleConfigFileOrBuffer, process.cwd(), { interactive, overwrite, merge }, extraEnvVars)
+    return config
   }
 
-  async run () {
-    const { flags, args } = this.parse(Use)
+  async downloadConsoleConfigToBuffer (consoleCLI, config, supportedServices) {
+    const workspaceConfig = await consoleCLI.getWorkspaceConfig(
+      config.org.id,
+      config.project.id,
+      config.workspace.id,
+      supportedServices
+    )
+    return Buffer.from(JSON.stringify(workspaceConfig))
+  }
 
-    if (args.config_file_path) {
-      return this.importConfigFile(args.config_file_path, flags)
+  async finalLogMessage (consoleConfig) {
+    const config = { org: consoleConfig.project.org, project: consoleConfig.project, workspace: consoleConfig.project.workspace }
+    const configString = this.configString(config)
+    this.log(chalk.green(chalk.bold(
+      `${EOL}✔ Successfully imported configuration for:${EOL}${configString}`
+    )))
+  }
+
+  equalSets (setA, setB) {
+    if (setA.size !== setB.size) {
+      return false
     }
-    const file = await this.useConsoleConfig(flags)
-    if (file) {
-      const config = this.importConfigFile(file, flags)
-      // delete file only if it was downloaded
-      fs.unlinkSync(file)
-      return config
+    for (const a of setA) {
+      if (!setB.has(a)) {
+        return false
+      }
     }
+    return true
   }
 }
 
-Use.description = `Import an Adobe I/O Developer Console configuration file
+Use.description = `Import an Adobe Developer Console configuration file
 `
 
 Use.flags = {
   ...BaseCommand.flags,
   overwrite: flags.boolean({
-    description: 'Overwrite any .aio and .env files during import of the Adobe I/O Developer Console configuration file',
-    char: 'w',
```

Comment:
yes we discussed this internally: omitting -w and -m because -w will be used for -workspace and the alternative -o might be used for the more standard --output flag in the future. That's the breaking change
+ --merge and --overwrite are not being used much meaning we can spare the short chars

Also --merge can be set via --no-input

Reply:
SGTM